places/berlin.ini

```diff
@@ -81,20 +81,12 @@ location_path = a/div[2]/span:(?<=\|).*
 rooms_path = a/div[2]/ul[@class='article__properties']/li[1]/span
 
 [company:www.berlinhaus.com]
-url = https://www.berlinhaus.com/suchergebnisse/?filter_search_action[]=wohnen&advanced_city=berlin&zimmeranzahl-ab=1
-ad_path = .//div[@class='col-md-12 listing_wrapper']
-url_path = div/h4/a/@href
-title_path = div/h4/a
-location_path = div/div[2]/a[1]
-rooms_path = div/div[4]/div[2]
-
-[company:www.gesobau.de]
```

Comment:
Yeah, the updated GESOBAU page seems to be a little confusing, but below the huge register section flat ads are fortunately still listed. :sweat_smile: Parsing fails because the HTML was slightly restructured.